torch/csrc/xpu/Module.cpp

```diff
@@ -19,7 +19,7 @@ static PyObject* THXPModule_getArchFlags(PyObject* self, PyObject* noargs) {
   HANDLE_TH_ERRORS
 #ifdef XPU_ARCH_FLAGS
   static const char* flags = C10_STRINGIZE(XPU_ARCH_FLAGS);
-  return THPUtils_packString(flags);
+  return THPUtils_packString(strcmp(flags, "\"\"") == 0 ? "" : flags);
```

Comment:
Set the value back from `""` to an empty string to avoid `torch._C._xpu_getArchFlags()` function returns `""` as the arch flag.